provisioning/stream/aws.sh

```diff
@@ -1,27 +0,0 @@
-#!/usr/bin/env bash
-set -Eeuo pipefail
-
-# Prevent direct run of the script
```

Comment:
There is a lot of modifications, so this commit removes old definitions, and following commits add each service part, step by step.